### ISSUE TITLE
[bug] [comfy_mtb] | INFO -> Some nodes failed to load

### ISSUE THREAD

**Issue by Franks001** [melMass/comfy_mtb#81]
### Describe the bug

After installing the plugin and restarting, some errors were found，
[comfy_mtb] | INFO -> Some nodes failed to load:
        Failed to import module faceswap because ImportError: cannot import name 'mesh_core_cython' from 'insightface.thirdparty.face3d.mesh.cython' (unknown location)
        Failed to import module image_interpolation because If you need help, create an issue at https://github.com/tensorflow/tensorflow/issues and include the entire stack trace above this error message.

### Platform and versions

```sh
.Windows10
.ComfyUI_windows_portable
```

### Console output

```sh
E:\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build --port 8192
** ComfyUI start up time: 2023-08-28 08:18:37.389336

Prestartup times for custom nodes:
   0.0 seconds: E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 11264 MB, total RAM 32725 MB
xformers version: 0.0.21
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 2080 Ti : cudaMallocAsync
Using xformers cross attention
Adding extra search path checkpoints E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/Stable-diffusion
Adding extra search path configs E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/Stable-diffusion
Adding extra search path vae E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/VAE
Adding extra search path loras E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/Lora
Adding extra search path loras E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/LyCORIS
Adding extra search path upscale_models E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/ESRGAN
Adding extra search path upscale_models E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/RealESRGAN
Adding extra search path upscale_models E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/SwinIR
Adding extra search path embeddings E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\embeddings
Adding extra search path hypernetworks E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/hypernetworks
Adding extra search path controlnet E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/ControlNet
Adding extra search path clip_vision E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/clip_vision
Adding extra search path sams E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/sams
Adding extra search path seecoders E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/seecoders
Adding extra search path ultralytics E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/ultralytics
Adding extra search path gligen E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/gligen
Adding extra search path IPAdapter E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/IPAdapter_models
Adding extra search path IPAdapter E:\AI\SD\webui\QiuYe\sd-webui-aki-v4.2\models/IPAdapter_sdxl_models
### Loading: ComfyUI-Manager (V0.26.2)
### ComfyUI Revision: 1348 [412596d3]
E:\ComfyUI_windows_portable\python_embeded\lib\site-packages\torchvision\transforms\functional_tensor.py:5: UserWarning: The torchvision.transforms.functional_tensor module is deprecated in 0.15 and will be **removed in 0.17**. Please don't rely on it. You probably just need to use APIs in torchvision.transforms.functional or in torchvision.transforms.v2.functional.
  warnings.warn(
[comfy_mtb] | INFO -> Some nodes failed to load:
        Failed to import module faceswap because ImportError: cannot import name 'mesh_core_cython' from 'insightface.thirdparty.face3d.mesh.cython' (unknown location)
        Failed to import module image_interpolation because If you need help, create an issue at https://github.com/tensorflow/tensorflow/issues and include the entire stack trace above this error message.

Check that you properly installed the dependencies.
If you think this is a bug, please report it on the github page (https://github.com/melMass/comfy_mtb/issues)
[comfy_mtb] | INFO -> Loaded the following nodes:
        Animation Builder (mtb): Convenient way to manage basic animation maths at the core of many of my workflows
        Smart Step (mtb): Utils to control the steps start/stop of the KAdvancedSampler in percentage
        Styles Loader (mtb): Load csv files and populate a dropdown from the rows (à la A111)
        Interpolate Clip Sequential (mtb): -
        Bbox From Mask (mtb): From a mask extract the bounding box
        Bbox (mtb): The bounding box (BBOX) custom type used by other nodes
        Crop (mtb): Crops an image and an optional mask to a given bounding box
        Uncrop (mtb): Uncrops an image to a given bounding box
        Debug (mtb): Experimental node to debug any Comfy values, support for more types and widgets is planned
        Save Tensors (mtb): Save torch tensors (image, mask or latent) to disk, useful to debug things outside comfy
        Deep Bump (mtb): Normal & height maps generation from single pictures
        Restore Face (mtb): Uses GFPGan to restore faces
        Load Face Enhance Model (mtb): Loads a GFPGan or RestoreFormer model for face enhancement.
        Qr Code (mtb): Basic QR Code generator
        Unsplash Image (mtb): Unsplash Image given a keyword and a size
        Text To Image (mtb): Utils to convert text to image using a font
        String Replace (mtb): Basic string replacement
        Fit Number (mtb): Fit the input float using a source and target range
        Get Batch From History (mtb): Very experimental node to load images from the history of the server.
        Any To String (mtb): Tries to take any input and convert it to a string
        Color Correct (mtb): Various color correction methods
        Image Compare (mtb): Compare two images and return a difference image
        Blur (mtb): Blur an image using a Gaussian filter.
        Mask To Image (mtb): Converts a mask (alpha) to an RGB image with a color and background
        Colored Image (mtb): Constant color image of given size
        Image Premultiply (mtb): Premultiply image with mask
        Image Resize Factor (mtb): Extracted mostly from WAS Node Suite, with a few edits (most notably multiple image support) and less features.
        Save Image Grid (mtb): Save all the images in the input batch as a grid of images.
        Load Image From Url (mtb): Load an image from the given URL
        Save Gif (mtb): Save the images from the batch as a GIF
        Export With Ffmpeg (mtb): Export with FFmpeg (Experimental)
        Latent Lerp (mtb): Linear interpolation (blend) between two latent vectors
        Image Remove Background Rembg (mtb): Removes the background from the input using Rembg.
        Float To Number (mtb): Node addon for the WAS Suite. Converts a "comfy" FLOAT to a NUMBER.
        Int To Bool (mtb): Basic int to bool conversion
        Int To Number (mtb): Node addon for the WAS Suite. Converts a "comfy" INT to a NUMBER.
        Transform Image (mtb): Save torch tensors (image, mask or latent) to disk, useful to debug things outside comfy
        Load Image Sequence (mtb): Load an image sequence from a folder. The current frame is used to determine which image to load.
        Save Image Sequence (mtb): Save an image sequence to a folder. The current frame is used to determine which image to save.

Import times for custom nodes:
   0.0 seconds: E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\Derfuu_ComfyUI_ModdedNodes
   0.2 seconds: E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager
   1.0 seconds: E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfy_mtb

Starting server

To see the GUI go to: http://127.0.0.1:8192
FETCH DATA from: E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json
[comfy_mtb] | WARNING -> No Face Restore checkpoints found at E:\ComfyUI_windows_portable\ComfyUI\models\face_restore (if you've used mtb before these checkpoints were saved in upscale_models before)
[comfy_mtb] | WARNING -> For now we fallback to upscale_models but this will be removed in a future version
```

### Additional context

The following nodes are missing when loading case 01-faceswap.json
Load Face Swap Model (mtb)
Load Face Analysis Model (mtb)
Face Swap (mtb)

**Comment by Linaghan34** [comment]
请问inswapper_128应该放哪个model 文件夹

**Comment by Franks001** [comment]
> 请问inswapper_128应该放哪个model 文件夹

目前我所有的的模型都在这个路径：
Currently all my models are in this path：
E:\ComfyUI_windows_portable\ComfyUI\models\upscale_models

**Comment by melMass** [comment]
I will look at it but this one is a first. Are you using Comfy nightly?

**Comment by Franks001** [comment]
> I will look at it but this one is a first. Are you using Comfy nightly?
Thank you. I have tried to solve the problem myself, as follows,
1. Removed this path E:  ComfyUI_ Windows_ Portable  Python_ The following files in embedded  Lib  site packages:
Insightface package and TensorFlow package
2. Start Python on Windows 10 and install Insightface and TensorFlow,
3. Copy Insightface and TensorFlow to the virtual environment
4. Start run_ Nvidia_ Gpu.bat, the previous error has disappeared, and the nodes were not lost when loading four cases.
